tests/functional/test_to_idp_interstitial.py

```diff
@@ -0,0 +1,38 @@
+import time
+
+from flask import url_for
+import pytest
+
+
+@pytest.mark.usefixtures('live_server')
+class When_on_to_idp_interstitial(object):
+
+    @pytest.fixture(autouse=True)
+    def setup_page(
+            self,
+            browser,
+            email_address,
+            submit_known_email_address):
+        submit_known_email_address(email_address)
+        browser.find_by_css('form button').click()
+
+    def it_goes_to_the_idp_after_clicking_continue(
+            self, browser, config, responses):
+
+        browser.find_by_css(
+            '#content>div.pagination>ul>'
+            'li>a>span.pagination-text>span').click()
+
+        assert browser.url.startswith(config.get('OIDC_CLIENT')['issuer'])
```

Comment:
This causes the tests to make a request to `http://example.com` when they are run, which unfortunately makes the test dependent on having access to the Internet to run (and that `http://example.com` is available).

The problem is that the test is non-deterministic - if there are connectivity issues when the test is run, it could produce a false negative result.